<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditModeSelectorWidget</class>
 <widget class="QWidget" name="EditModeSelectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>110</width>
    <height>26</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EditModeSelector</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <item>
       <widget class="QToolButton" name="m_moveButton">
        <property name="toolTip">
         <string>Move triangle</string>
        </property>
        <property name="text">
         <string>Move Tringle</string>
        </property>
        <property name="icon">
         <iconset resource="../qosmic.qrc">
          <normaloff>:/icons/silk/arrow_updown.xpm</normaloff>:/icons/silk/arrow_updown.xpm</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_rotateButton">
        <property name="toolTip">
         <string>Rotate triangle</string>
        </property>
        <property name="text">
         <string>Move Tringle</string>
        </property>
        <property name="icon">
         <iconset resource="../qosmic.qrc">
          <normaloff>:/icons/silk/arrow_rotate_clockwise.xpm</normaloff>:/icons/silk/arrow_rotate_clockwise.xpm</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_scaleButton">
        <property name="toolTip">
         <string>Scale triangle</string>
        </property>
        <property name="text">
         <string>Move Tringle</string>
        </property>
        <property name="icon">
         <iconset resource="../qosmic.qrc">
          <normaloff>:/icons/silk/arrow_inout.xpm</normaloff>:/icons/silk/arrow_inout.xpm</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_flipButton">
        <property name="toolTip">
         <string>Flip triangle</string>
        </property>
        <property name="text">
         <string>Move Tringle</string>
        </property>
        <property name="icon">
         <iconset resource="../qosmic.qrc">
          <normaloff>:/icons/silk/shape_flip_horizontal.xpm</normaloff>:/icons/silk/shape_flip_horizontal.xpm</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qosmic.qrc"/>
 </resources>
 <connections/>
</ui>
